### ISSUE TITLE
babel warning when running locally: Though the "loose" option was set to "false" it will not be used

### ISSUE THREAD

**Issue by eyalkenig** [Shopify/argo-admin-template#99]
Hi,
Thanks for this interesting new ecosystem you've built :)

when running the app (with `shopify serve` or with `npm run server`) - there is a warning that is printed A LOT of times:
```
> argo-admin-cli server --entry="index.tsx" --port=39351 --type=PRODUCT_SUBSCRIPTION
....
Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-methods since the "loose" mode option was set to "true" for @babel/plugin-proposal-class-properties.
The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding
	["@babel/plugin-proposal-private-methods", { "loose": true }]
```
![image](https://user-images.githubusercontent.com/11048153/117611723-af443800-b16c-11eb-9588-8f4f2d2b192f.png)

It's still working, but pretty annoying :)

**Comment by vividviolet** [comment]
@eyalkenig this should have been fixed in `@shopify/argo-admin-cli` version `0.11.0`. Can you try installing it and see if the issue is still there?
```
npm install @shopify/argo-admin-cli@0.11.0
```

**Comment by eyalkenig** [comment]
looks good, thanks :+1:
